BuildToolBuilder/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace BuildToolBuilder
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            throw new Exception("This program should not be executed. It is used as project reference to compile the BuildTool project.");
```

Comment:
Maybe change `Exception` to `InvalidOperationException` or `NotSupportedException`